src/asm/heroes2_imports.inc

```diff
@@ -148,6 +148,8 @@ IMPORT_?SetRandomHeroArmies@game@@QAEXHH@Z = 1
 ?SetRandomHeroArmies@game@@QAEXHH@Z_clone EQU ?SetRandomHeroArmies_orig@game@@QAEXHH@Z
 IMPORT_?GiveTroopsToNeutralTown@game@@QAEXH@Z = 1
 ?GiveTroopsToNeutralTown@game@@QAEXH@Z_clone EQU ?GiveTroopsToNeutralTown_orig@game@@QAEXH@Z
+IMPORT_?ProcessOnMapHeroes@game@@QAEXXZ = 1
+?ProcessOnMapHeroes@game@@QAEXXZ_clone EQU ?ProcessOnMapHeroes_orig@game@@QAEXXZ
```

Comment:
You're not calling ProcessOnMapHeroes_orig, are you? If so, then this is not needed. Removing this line helps us distinguish between functions that we replace entirely, vs. merely wrap.

Reply:
Yeah, take a few minutes to do so.